DigitalOffice/Tests/UserServiceTests/Validators/EditUserRequestValidatorTests.cs

```diff
@@ -0,0 +1,118 @@
+﻿using FluentValidation;
+using FluentValidation.TestHelper;
+using NUnit.Framework;
+using LT.DigitalOffice.UserService.Models;
+using LT.DigitalOffice.UserService.Validators;
+using System;
+
+namespace LT.DigitalOffice.UserServiceUnitTests.Validators
+{
+    public class EditUserRequestValidatorTests
```

Comment:
You are checking not all rules. Check CreateUserRequestValidatorTests in refactoring pull request.

Reply:
Fixed